encoding.bs

```diff
@@ -1266,12 +1264,12 @@ method, when invoked, must run these steps:
    <li><p>Let <var>token</var> be the result of <a>reading</a> from <a for=TextDecoder>stream</a>.
 
    <li>
-    <p>If <var>token</var> is <a>end-of-stream</a> and the <a for=TextDecoder>do not flush flag</a>
-    is set, then return <var>output</var>,
+    <p>If <var>token</var> is <a>end-of-stream</a> and <a for=TextDecoder>do not flush</a>
+    is true, then return <var>output</var>,
     <a lt="serialize stream" for=TextDecoderCommon>serialized</a>.
 
-    <p class=note>The way streaming works is to not handle <a>end-of-stream</a> here when the
-    <a for=TextDecoder>do not flush flag</a> is set and to not unset that flag. That way in a
+    <p class=note>The way streaming works is to not handle <a>end-of-stream</a> here when
+    <a for=TextDecoder>do not flush</a> is true and to not set that slot to false. That way in a
```

Comment:
The literal translation of "to not unset that flag" would be "to not set that boolean to false", but since its definition doesn't explicitly mark it as a boolean, I wasn't sure whether it was appropriate. "Slot" might not be the right term here. "Associated value" perhaps?

Reply:
s/that slot/it/ would work? Or duplicate the term.